docs/source/aws/lambda.rst

```diff
@@ -80,11 +81,15 @@ have been defined to allow for easier policy writing as for the
         event: RunInstances
         ids: "responseElements.instancesSet.items[].instanceId"
 
+Refer to the `AWS execution modes documention
+<https://cloudcustodian.io/docs/aws/resources/aws-modes.html#cloudtrail>`_ for a
+list of configuration options for this execution mode: 
```

Comment:
`for a list of configuration options for this execution mode:` --> `for a list of other configurable options`